[CI] MixedClusterEsqlSpecIT test {topN.SortingOnSwappedFields SYNC} failing

**Build Scans:**
- [elasticsearch-periodic #4421 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/htk4pnyg3jqwy)
- [elasticsearch-periodic #4421 / 8.12.2_bwc](https://gradle-enterprise.elastic.co/s/xn6nyqhrgreo2)
- [elasticsearch-periodic #4415 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/zrpt6mch25qma)
- [elasticsearch-periodic #4411 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/rewo7wlow2bmg)
- [elasticsearch-periodic #4411 / 8.12.2_bwc](https://gradle-enterprise.elastic.co/s/kdsslew6i2ja6)
- [elasticsearch-periodic #4379 / 8.11.4_bwc](https://gradle-enterprise.elastic.co/s/yi2vbrtzg7iye)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:mixed-cluster:v8.12.2#javaRestTest" -Dtests.class="org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT" -Dtests.method="test {topN.SortingOnSwappedFields SYNC}" -Dtests.seed=48A1CBEFB3B1444A -Dtests.bwc=true -Dtests.locale=sd-Arab -Dtests.timezone=Etc/GMT-3 -Druntime.java=22
```

**Applicable branches:**
8.x

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupInput:(chainingSystem:HIERARCHICAL,controlStyle:twoLine,ignoreParentSettings:(ignoreFilters:!f,ignoreQuery:!f,ignoreTimerange:!f,ignoreValidations:!t),panels:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:task.keyword,grow:!t,id:'0c0c9cb8-ccd2-45c6-9b13-96bac4abc542',searchTechnique:wildcard,selectedOptions:!(),singleSelect:!t,title:'Gradle%20Task',width:medium),grow:!t,order:0,type:optionsListControl,width:small),'144933da-5c1b-4257-a969-7f43455a7901':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:name.keyword,grow:!t,id:'144933da-5c1b-4257-a969-7f43455a7901',searchTechnique:wildcard,selectedOptions:!('test%20%7BtopN.SortingOnSwappedFields%20SYNC%7D'),title:Test,width:medium),grow:!t,order:2,type:optionsListControl,width:medium),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(explicitInput:(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,enhancements:(),fieldName:className.keyword,grow:!t,id:'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850',searchTechnique:wildcard,selectedOptions:!('org.elasticsearch.xpack.esql.qa.mixed.MixedClusterEsqlSpecIT'),title:Suite,width:medium),grow:!t,order:1,type:optionsListControl,width:medium)))))

**Failure Message:**
```
org.elasticsearch.client.ResponseException: method [POST], host [http://[::1]:43427], URI [/_query?format=smile&pretty=true&error_trace=true], status line [HTTP/1.1 500 Internal Server Error]
:)
úerrorúroot_causeøútypeUnull_pointer_exceptionreasonàCannot invoke "org.elasticsearch.xpack.esql.planner.Layout$ChannelAndType.channel()" because "input" is nullüstack_traceäorg.elasticsearch.ElasticsearchException$1: Cannot invoke "org.elasticsearch.xpack.esql.planner.Layout$ChannelAndType.channel()" because "input" is null
	at org.elasticsearch.server@8.12.2/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:699)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:627)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.rest.RestResponse.build(RestResponse.java:180)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:140)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:111)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:55)
	at org.elasticsearch.xpack.esql.action.EsqlResponseListener.lambda$wrapWithLogging$1(EsqlResponseListener.java:168)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListener$2.onFailure(ActionListener.java:186)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.rest.action.RestCancellableNodeClient$1.onFailure(RestCancellableNodeClient.java:96)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:354)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:214)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$execute$1(ComputeService.java:144)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.acceptException(ActionListenerImplementations.java:186)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$DelegatingResponseActionListener.onFailure(ActionListenerImplementations.java:191)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener.onFailure(ActionListenerImplementations.java:308)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:354)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:27)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:73)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListener$4.onFailure(ActionListener.java:354)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.RefCountingListener.finish(RefCountingListener.java:127)
	at org.elasticsearch.base@8.12.2/org.elasticsearch.core.AbstractRefCounted$1.closeInternal(AbstractRefCounted.java:118)
	at org.elasticsearch.base@8.12.2/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:70)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.RefCountingRunnable.close(RefCountingRunnable.java:112)
	at org.elasticsearch.base@8.12.2/org.elasticsearch.core.Releasables$3.close(Releasables.java:142)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.RefCountingListener$1.onFailure(RefCountingListener.java:158)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.SubscribableListener$FailureResult.complete(SubscribableListener.java:322)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.SubscribableListener.tryComplete(SubscribableListener.java:234)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.SubscribableListener.setResult(SubscribableListener.java:275)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.SubscribableListener.onFailure(SubscribableListener.java:178)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler.closeInternal(ExchangeSourceHandler.java:244)
	at org.elasticsearch.base@8.12.2/org.elasticsearch.core.AbstractRefCounted.decRef(AbstractRefCounted.java:70)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$PendingInstances.finishInstance(ExchangeSourceHandler.java:267)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.onSinkComplete(ExchangeSourceHandler.java:208)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.onSinkFailed(ExchangeSourceHandler.java:202)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:62)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListener$2.onFailure(ActionListener.java:186)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleException(ActionListenerResponseHandler.java:54)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.handleSendRequestException(TransportService.java:857)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:851)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
	at org.elasticsearch.compute.operator.exchange.ExchangeService$TransportRemoteSink.fetchPageAsync(ExchangeService.java:271)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.fetchPage(ExchangeSourceHandler.java:159)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$1.doRun(ExchangeSourceHandler.java:234)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.NullPointerException: Cannot invoke "org.elasticsearch.xpack.esql.planner.Layout$ChannelAndType.channel()" because "input" is null
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planProject(LocalExecutionPlanner.java:536)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:199)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planOutput(LocalExecutionPlanner.java:280)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:227)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:170)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:312)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$execute$7(ComputeService.java:206)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:32)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:49)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.handleResponse(TransportService.java:1709)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1425)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1524)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel$1.doRun(TransportService.java:1501)
	... 7 more
	Suppressed: org.elasticsearch.transport.SendRequestTransportException: [test-cluster-0][127.0.0.1:35017][indices:data/read/esql/data]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:851)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
		at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$runComputeOnRemoteNodes$11(ComputeService.java:269)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:49)
		... 10 more
	Caused by: org.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
		... 16 more
	Suppressed: org.elasticsearch.transport.SendRequestTransportException: [test-cluster-0][127.0.0.1:35017][internal:data/read/esql/exchange]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:851)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
		at org.elasticsearch.compute.operator.exchange.ExchangeService$TransportRemoteSink.fetchPageAsync(ExchangeService.java:271)
		at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.fetchPage(ExchangeSourceHandler.java:159)
		at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$1.doRun(ExchangeSourceHandler.java:234)
		... 7 more
	Caused by: org.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
		... 12 more
üûùBUnull_pointer_exceptionCàCannot invoke "org.elasticsearch.xpack.esql.planner.Layout$ChannelAndType.channel()" because "input" is nullüDäjava.lang.NullPointerException: Cannot invoke "org.elasticsearch.xpack.esql.planner.Layout$ChannelAndType.channel()" because "input" is null
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planProject(LocalExecutionPlanner.java:536)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:199)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.planOutput(LocalExecutionPlanner.java:280)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:227)
	at org.elasticsearch.xpack.esql.planner.LocalExecutionPlanner.plan(LocalExecutionPlanner.java:170)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.runCompute(ComputeService.java:312)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$execute$7(ComputeService.java:206)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.support.ContextPreservingActionListener.onResponse(ContextPreservingActionListener.java:32)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:49)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$UnregisterChildTransportResponseHandler.handleResponse(TransportService.java:1709)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1425)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1524)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel$1.doRun(TransportService.java:1501)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
	Suppressed: org.elasticsearch.transport.SendRequestTransportException: [test-cluster-0][127.0.0.1:35017][indices:data/read/esql/data]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:851)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
		at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$runComputeOnRemoteNodes$11(ComputeService.java:269)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:49)
		... 10 more
	Caused by: org.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
		... 16 more
	Suppressed: org.elasticsearch.transport.SendRequestTransportException: [test-cluster-0][127.0.0.1:35017][internal:data/read/esql/exchange]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:851)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
		at org.elasticsearch.compute.operator.exchange.ExchangeService$TransportRemoteSink.fetchPageAsync(ExchangeService.java:271)
		at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.fetchPage(ExchangeSourceHandler.java:159)
		at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$1.doRun(ExchangeSourceHandler.java:234)
		... 7 more
	Caused by: org.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
		at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
		... 12 more
üsuppressedøúBWtask_cancelled_exceptionCdparent task was cancelled [cancelled]Dàorg.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
	at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
	at org.elasticsearch.xpack.esql.plugin.ComputeService.lambda$runComputeOnRemoteNodes$11(ComputeService.java:269)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$ResponseWrappingActionListener.onResponse(ActionListenerImplementations.java:236)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerImplementations$MappedActionListener.onResponse(ActionListenerImplementations.java:95)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.action.ActionListenerResponseHandler.handleResponse(ActionListenerResponseHandler.java:49)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$ContextRestoreResponseHandler.handleResponse(TransportService.java:1425)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:1524)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService$DirectResponseChannel$1.doRun(TransportService.java:1501)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
üûúBWtask_cancelled_exceptionCdparent task was cancelled [cancelled]Dàorg.elasticsearch.tasks.TaskCancelledException: parent task was cancelled [cancelled]
	at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager$CancellableTaskHolder.registerChildConnection(TaskManager.java:666)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.tasks.TaskManager.registerChildConnection(TaskManager.java:362)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendRequest(TransportService.java:831)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:911)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.transport.TransportService.sendChildRequest(TransportService.java:888)
	at org.elasticsearch.compute.operator.exchange.ExchangeService$TransportRemoteSink.fetchPageAsync(ExchangeService.java:271)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$RemoteSinkFetcher.fetchPage(ExchangeSourceHandler.java:159)
	at org.elasticsearch.compute.operator.exchange.ExchangeSourceHandler$1.doRun(ExchangeSourceHandler.java:234)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.TimedRunnable.doRun(TimedRunnable.java:33)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.ThreadContext$ContextPreservingAbstractRunnable.doRun(ThreadContext.java:983)
	at org.elasticsearch.server@8.12.2/org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:26)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
üûùûstatus$¨û
```

**Issue Reasons:**
- [8.x] 4 consecutive failures in step 8.11.4_bwc
- [8.x] 6 failures in test test {topN.SortingOnSwappedFields SYNC} (2.7% fail rate in 219 executions)
- [8.x] 4 failures in step 8.11.4_bwc (100.0% fail rate in 4 executions)
- [8.x] 2 failures in step 8.12.2_bwc (66.7% fail rate in 3 executions)
- [8.x] 4 failures in pipeline elasticsearch-periodic (100.0% fail rate in 4 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/date.csv-spec
@@ -367,7 +367,7 @@ date1:date               | dd_ms:integer
 2023-12-02T11:00:00.000Z | 1000
 ;
 
-evalDateDiffMonthAsWhole0Months
+evalDateDiffMonthAsWhole0Months#[skip:-8.14.1, reason:omitting millis/timezone not allowed before 8.14]
 
 ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-01-01T00:00:00")
 | EVAL msecs=DATE_DIFF("milliseconds", from, to), months=DATE_DIFF("month", from, to)
@@ -378,7 +378,7 @@ ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-01-01T00:
 
 ;
 
-evalDateDiffMonthAsWhole1Month
+evalDateDiffMonthAsWhole1Month#[skip:-8.14.1, reason:omitting millis/timezone not allowed before 8.14]
 
 ROW from=TO_DATETIME("2023-12-31T23:59:59.999Z"), to=TO_DATETIME("2024-02-01T00:00:00")
 | EVAL secs=DATE_DIFF("seconds", from, to), months=DATE_DIFF("month", from, to)

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/topN.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/topN.csv-spec
@@ -135,7 +135,7 @@ null              |Swan             |-8.46                       |-8.46         
 Sanjiv            |Zschoche         |[-7.67, -3.25]              |[-3.25, -7.67]               |[-3, -8]             |10053
 ;
 
-sortingOnSwappedFields
+sortingOnSwappedFields#[skip:-8.13.3, reason:fixed in 8.13]
 FROM employees
 | EVAL name = last_name, last_name = first_name, first_name = name
 | WHERE first_name > "B" AND last_name IS NOT NULL
@@ -157,7 +157,7 @@ Brattka      | Charlene          | Brattka
 Bridgland    | Patricio          | Bridgland
 ;
 
-sortingOnSwappedFieldsNoKeep
+sortingOnSwappedFieldsNoKeep#[skip:-8.13.3, reason:fixed in 8.13]
 // Note that this test requires all fields to be returned in order to test a specific code path in physical planning
 FROM employees
 | EVAL name = first_name, first_name = last_name, last_name = name

